api/routes_rtfsp.go

```diff
@@ -530,105 +531,118 @@ func (api *API) publishDetailedIPNSToHostedIPFSNetwork(c *gin.Context) {
 }
 
 // CreateHostedIPFSNetworkEntryInDatabase is used to create an entry in the database for a private ipfs network
-// TODO: make bootstrap peers and related config optional
 func (api *API) createHostedIPFSNetworkEntryInDatabase(c *gin.Context) {
-	// lock down as admin route for now
+	// retrieve fields
 	username := GetAuthenticatedUserFromContext(c)
-	if err := api.validateAdminRequest(username); err != nil {
-		FailNotAuthorized(c, eh.UnAuthorizedAdminAccess)
-		return
-	}
-	forms := api.extractPostForms(c, "network_name", "api_url", "swarm_key")
-	if len(forms) == 0 {
-		return
-	}
-
-	bPeers, exists := c.GetPostFormArray("bootstrap_peers")
-	if !exists {
-		FailWithMissingField(c, "bootstrap_peers")
-		return
-	}
-	nodeAddresses, exists := c.GetPostFormArray("local_node_addresses")
+	networkName, exists := c.GetPostForm("network_name")
 	if !exists {
-		FailWithMissingField(c, "local_node_addresses")
+		FailWithBadRequest(c, "network_name")
```

Comment:
In order to communicate the error is due to a missing field shouldn't it be:
```suggestion
		FailWithMissingField(c, "network_name")
```